google/cloud/forseti/services/dao.py

```diff
@@ -545,7 +639,29 @@ def scanner_iter(cls, session, resource_type,
         @classmethod
         def explain_denied(cls, session, member_name, resource_type_names,
                            permission_names, role_names):
-            """Provide information how to grant access to a member."""
+            """Provide information how to grant access to a member if such
```

Comment:
The doc string here should be one physical line that's a summary, followed by a blank line, and then more detailed description.
https://google.github.io/styleguide/pyguide.html#Comments

Reply:
Got it